src/App.js

```diff
@@ -64,16 +65,17 @@ function App() {
               />
             ))}
 
-            <Route exact path="/languages" component={Languages} />
-            <Route exact path="/frameworks" component={Frameworks} />
-            <Route exact path="/tags" component={Tags} />
-            <Route exact path="/softwares" component={Softwares} />
-            <Route exact path="/databases" component={Databases} />
-            <Route exact path="/" component={Home}></Route>
-          </Switch>
-          <ScrollToTop />
-        </>
-      )}
+        <Route exact path="/contactus" component={ContactUs} />
+        <Route exact path="/languages" component={Languages} />
+        <Route exact path="/frameworks" component={Frameworks} />
+        <Route exact path="/tags" component={Tags} />
+        <Route exact path="/softwares" component={Softwares} />
+        <Route exact path="/development" component={Development} />
+        <Route exact path="/" component={Home}></Route>
+        
+      </Switch>
+      <ScrollToTop />
+
     </div>
```

Comment:
Expected corresponding JSX closing tag for <> (79:4)

  77 |       <ScrollToTop />
  78 |
> 79 |     </div>
     |     ^
  80 |   );
  81 | }
  82 |

Reply:
Kindly resolve the errors.